Easybuoy/storemanager
# #161531821 fix tests issues

#### What does this PR do?
Re-writing of tests to work with Postgres database.

#### Description of Task to be completed?
Change all tests using the mock database to using Postgres databtase.

#### How should this be manually tested?
After cloning the repo, cd into storemanager and run npm run test
Response should be all tests passing.

#### Any background context you want to provide?
None
